lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb

```diff
@@ -157,6 +157,10 @@ module ActiveRecord
       #       should validate_uniqueness_of(:slug).scoped_to(:journal_id)
       #     end
       #
+      # NOTE: This gem does not fully support testing uniqueness scoped to an
+      # array of associations. For a reference thread, please check this issue
+      # https://github.com/thoughtbot/shoulda-matchers/issues/814
+      #
```

Comment:
```suggestion
# Support for testing uniqueness validation scoped to an association is not available.
# For more information, please refer to https://github.com/thoughtbot/shoulda-matchers/issues/814
```
I think we can get rid of `gem` word as its contextual. Also i think array is not needed.
Does this change makes sense?

Reply:
Hey @KapilSachdev I completely agree with the first part about removing `gem`. But the array part is actually the point of this PR. We can validate scoped to an association (using attribute name, not object), but when an object is uniquely scoped to more than one object, we can't really validate it. It is possible to test with scope to a list of attributes, but it is unclear how this would behave in multiple scenarios.

The solutions proposed on #814 we not as simple as I thought it could be when I first created the issue, and manually testing it (without shoulda) would be easier to read. Because of this, it was decided to add this specific comment to make clear this limitation.

I'll change the text and update PR. Thanks for suggestion